share/ert/workflows/jobs/internal/scripts/enkf_scaling_job.py

```diff
@@ -0,0 +1,29 @@
+import yaml
+import collections
+
+from res.enkf import ErtScript
+from res.enkf.jobs.scaling import scaling_job
+
+
+class EnkfScalingJob(ErtScript):
+
+    def run(self, job_config_file):
+
+        user_config = load_yaml(job_config_file)
+
```

Comment:
do we need the empty lines?

Reply:
No we don't, I've removed them, you're right, easier to read that way.